dali/python/backend_impl.cc

```diff
@@ -330,11 +330,12 @@ void ExposeTensorList(py::module &m) {
       ----------
       )code")
       .def("__getitem__",
-        [](TensorList<CPUBackend> &t, Index id) -> std::unique_ptr<Tensor<CPUBackend>> {
-          return TensorListGetItemImpl(t, id);
+        [](TensorList<CPUBackend> &t, Index i) -> std::unique_ptr<Tensor<CPUBackend>> {
+          return TensorListGetItemImpl(t, i);
         },
+      "i"_a,
       R"code(
-      Returns a tensor at given position in the list.
+      Returns a tensor at given position `i` in the list.
```

Comment:
How about naming it `pos`? This would be better IMHO.

Plus, I think there should be double-back-tick around "i"

Reply:
`i` looks very much like C API docs. In case of [numpy ](https://docs.scipy.org/doc/numpy-1.15.0/reference/generated/numpy.ndarray.__getitem__.html)I see they use `key`.
I'm good with both options.

Reply:
In case of python docs there is an explanation using `i`:
```
then x[i] is roughly equivalent to type(x).__getitem__(x, i)
```
But the member function is documented with `key`.
https://docs.python.org/3.7/reference/datamodel.html#object.__getitem__

Is the key clear enough? I don't think I've ever seen a `key` being related to arrays.